src/todo-list-kata/day_01/kata.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+export default class TodoList extends React.Component {
```

Comment:
Better to have the same approach, as here
https://github.com/Alex-Diez/react-js-tdd-katas/pull/1/files#diff-14b1e33d5bf5649597cdc0e4f684daddR6